### PR TITLE
hyperliquid: add ws pro

### PR DESCRIPTION
**Pull request by sc0Vu**
DOC: https://hyperliquid.gitbook.io/hyperliquid-docs/for-developers/api/websocket#connection

TODO:
- [x] watchTrades
- [x] watchOrderBook
- [x] watchOHLCV
- [x] watchOrders
- [x] watchMyTrades

### CODE REVIEW

File: skip-tests.json [hunk 1]
```diff
@@ -1674,6 +1674,7 @@
                 "deposit": "not provided",
                 "withdraw": "not provided"
             }
-        }
+        },
+        "skipWs": true
```
Comment (carlosmiei) [inline skip-tests.json:1678]: @sc0Vu why are we skipping it?

Comment (sc0Vu) [reply skip-tests.json]: @carlosmiei because of the currency not found

Comment (carlosmiei) [reply skip-tests.json]: @sc0Vu We can make granular skips instead of skipping everything

**Comment by carlosmiei** [comment]
DEMO
```
p hyperliquid watchMyTrades  --sandbox
Python v3.11.7
CCXT v4.2.65
hyperliquid.watchMyTrades()
[{'amount': 0.04,
  'cost': 118.54,
  'datetime': '2024-02-24T14:22:32.393Z',
  'fee': {'cost': 0.041489, 'currency': 'USDC'},
  'fees': [{'cost': 0.041489, 'currency': 'USDC'}],
  'id': '59129319050113',
  'info': {'cloid': None,
           'closedPnl': '-0.0172',
           'coin': 'ETH',
           'crossed': True,
           'dir': 'Close Short',
           'fee': '0.041489',
           'hash': '0x48d12038cfd429417ac3040a984db7010c00fbd9e5fb94d5e8627e279deb2b95',
           'liquidationMarkPx': None,
           'oid': 6236360927,
           'px': '2963.5',
           'side': 'B',
           'startPosition': '-0.04',
           'sz': '0.04',
           'tid': 59129319050113,
           'time': 1708784552393},
  'order': None,
  'price': 2963.5,
  'side': 'buy',
  'symbol': 'ETH/USDC:USDC',
  'takerOrMaker': None,
  'timestamp': 1708784552393,
  'type': None},
 {'amount': 0.2,
  'cost': 220.0,
  'datetime': '2024-02-28T17:32:21.609Z',
  'fee': {'cost': 0.022, 'currency': 'USDC'},
  'fees': [{'cost': 0.022, 'currency': 'USDC'}],
  'id': '78441605493485',
  'info': {'cloid': None,
           'closedPnl': '0.0',
           'coin': 'ETH',
           'crossed': False,
           'dir': 'Open Long',
           'fee': '0.022',
           'hash': '0xd53d85c92584a07d96b8040ab529dd0000d3c35dd601be7d06d53a3e6c87c6c5',
           'liquidationMarkPx': None,
           'oid': 6270936770,
           'px': '1100.0',
           'side': 'B',
           'startPosition': '0.0',
           'sz': '0.2',
           'tid': 78441605493485,
           'time': 1709141541609},
  'order': None,
  'price': 1100.0,
  'side': 'buy',
  'symbol': 'ETH/USDC:USDC',
  'takerOrMaker': None,
  'timestamp': 1709141541609,
  'type': None},
```
```

➜  ccxt git:(add-hyperliquid-ws) ✗ p hyperliquid watchOrders "SOL/USDC:USDC" --sandbox
Python v3.11.7
CCXT v4.2.65
hyperliquid.watchOrders(SOL/USDC:USDC)
[{'amount': 0.2,
  'average': None,
  'clientOrderId': None,
  'cost': None,
  'datetime': '2024-03-13T12:25:24.478Z',
  'fee': None,
  'fees': [],
  'filled': None,
  'id': '7594629343',
  'info': {'order': {'coin': 'SOL',
                     'limitPx': '50.0',
                     'oid': 7594629343,
                     'origSz': '0.2',
                     'side': 'B',
                     'sz': '0.2',
                     'timestamp': 1710332724478},
           'status': 'open',
           'statusTimestamp': 1710332724478},
  'lastTradeTimestamp': None,
  'lastUpdateTimestamp': None,
  'postOnly': None,
  'price': 50.0,
  'reduceOnly': None,
  'remaining': None,
  'side': 'buy',
  'status': 'open',
  'stopLossPrice': None,
  'stopPrice': None,
  'symbol': 'SOL/USDC:USDC',
  'takeProfitPrice': None,
  'timeInForce': None,
  'timestamp': 1710332724478,
  'trades': [],
  'triggerPrice': None,
  'type': None}]
[{'amount': 0.2,
  'average': None,
  'clientOrderId': None,
  'cost': None,
  'datetime': '2024-03-13T12:26:00.061Z',
  'fee': None,
  'fees': [],
  'filled': None,
  'id': '7594629343',
  'info': {'order': {'coin': 'SOL',
                     'limitPx': '50.0',
                     'oid': 7594629343,
                     'origSz': '0.2',
                     'side': 'B',
                     'sz': '0.2',
                     'timestamp': 1710332724478},
           'status': 'canceled',
           'statusTimestamp': 1710332760061},
  'lastTradeTimestamp': None,
  'lastUpdateTimestamp': None,
  'postOnly': None,
  'price': 50.0,
  'reduceOnly': None,
  'remaining': None,
  'side': 'buy',
  'status': 'canceled',
  'stopLossPrice': None,
  'stopPrice': None,
  'symbol': 'SOL/USDC:USDC',
  'takeProfitPrice': None,
  'timeInForce': None,
  'timestamp': 1710332760061,
  'trades': [],
  'triggerPrice': None,
  'type': None}]
```

**Pull request merged**